youtube_dl/extractor/adobepass.py

```diff
@@ -325,6 +320,11 @@
     'col080': {
         'name': 'Columbus Telephone'
     },
+    'Comcast_SSO': {
+        'name': 'Comcast XFINITY',
+        'username_field': 'user',
+        'password_field': 'passwd',
+    },
```

Comment:
I just ordered the dictionary/array by name.  Sorry if that causes any problems.